documentation/MIGRATION_GUIDE.md

```diff
@@ -84,6 +84,8 @@ if err != nil {
 log.Printf("Status code: %d", rawResponse.StatusCode)
 ```
 
+> NOTE: runtime is import from **github.com/Azure/azure-sdk-for-go/sdk/azcore/runtime**
```

Comment:
Shall we also update all our snippet to include the import section?
We could add some empty line and comments to separate the import and code lines

Reply:
People may only focus on the things they most care about. They might not notice this important information if we only put this here without code

Reply:
I've added import section for all non-system packages.